superset/connectors/sqla/models.py

```diff
@@ -1193,13 +1234,13 @@ def fetch_metadata(self, commit: bool = True) -> None:
         """Fetches the metadata for the table and merges it in"""
         try:
             table_ = self.get_sqla_table_object()
-        except Exception as ex:
-            logger.exception(ex)
-            raise Exception(
+        except Exception:
```

Comment:
`SQLAlchemyError` would be nice, but riskier

Reply:
Good point. Having been burned by weird exceptions from SQLA I'm slightly hesitant to do this, but at least we'd catch the clean SQLA exceptions with it and probably avoid the majority of exceptions. And we can always change it to `Exception` later if needed. I'll make the change.